src/app/clientPage.tsx

```diff
@@ -92,8 +93,7 @@ export default function ClientPage({
       localStorage.setItem("summaries", JSON.stringify([newData]));
     }
   };
-
-  if (isError) return <Error />;
+  if (isError) return <Error reset={forceClose} setIsError={setIsError} />;
```

Comment:
there's an issue where the loading states still appear if the state is reset through the error. We should make sure we are resetting those loading states as well.

Reply:
Sorry Rhut! The state should be reset internally not by the components. I have a hotfix out for that.

Reply:
Will remove this and merge